server/app.js

```diff
@@ -6,27 +6,18 @@ const bodyParser = require('body-parser');
 
 const path = require('path');
 
-const con = require('./database');
+const fetchAll = require('./model/index.js');
 
 app.use(express.static(path.join(__dirname, '/../client/dist')));
 app.use(bodyParser.json());
 
+app.get('/reviews', function(req, res) {
```

Comment:
@gorakong This endpoint does not match your listed endpoints in your [service plan](https://docs.google.com/document/d/19M5W_dSp8zcd9T9t4eLTKkGgW8KGT3Iwcu6RKzFmRxI).

Reply:
@jcelder Good catch--I have refactored my code to match the appropriate endpoint, & fixed the associated get request & responses.